projects/file-parsing/types-converting/json-struct.go

```diff
@@ -0,0 +1,33 @@
+package filesConverting
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Score struct {
+	Name       string
+	High_Score int32
+}
+
+func JsonToStruct(textByte []byte) []Score {
+	result := []Score{}
+	s := Score{}
+
+	textTrimmed := strings.Trim(string(textByte), "[]\n")
+	textSliced := strings.Split(string(textTrimmed), "\n")
+
+	for _, text := range textSliced {
+		text = strings.Trim(text, ",")
+		if string(text[0]) != "#" {
```

Comment:
We'd often invert this; write:
```
if string(text[0]) == "#" {
  continue
}
```

This avoids our code always getting more and more nested as we do more checks, and means we don't need to keep as many ideas in our head at the same time.